### PR TITLE
Feature: APPLICS-763/764: Material Changes

### PR DESCRIPTION
**Pull request by christopher-semmens**
## Describe your changes

https://pins-ds.atlassian.net/browse/APPLICS-763?atlOrigin=eyJpIjoiYjg4MmI1OTNmZmNkNDc5Zjg5NjdiODU4ZDY0MTBjMWYiLCJwIjoiaiJ9

https://pins-ds.atlassian.net/browse/APPLICS-764?atlOrigin=eyJpIjoiNzYwNzQ5MWZkOWE3NDM5OTliZmI2YWRlMTdmNDYyYmQiLCJwIjoiaiJ9

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/views/applications/components/case-form/case-form-layout.njk [hunk 1]
```diff
@@ -73,12 +74,12 @@
...
-  {% elif componentName == 'titleWelsh' %}
-    {{ title('titleWelsh', values, errors) }}
+	{% elif componentName == 'titleWelsh' %}
+		{{ title('titleWelsh', values, errors) }}
 	{% elif componentName == 'description' %}
 		{{ description('description', values, errors) }}
-  {% elif componentName == 'descriptionWelsh' %}
-    {{ description('descriptionWelsh', values, errors) }}
+	{% elif componentName == 'descriptionWelsh' %}
+		{{ description('descriptionWelsh', values, errors) }}
```
Comment (HadrianHughesKC) [inline apps/web/src/server/views/applications/components/case-form/case-form-layout.njk:82]: There's some parts of the diff that are just changing tabs to spaces. Not a big deal but worth seeing if your text editor can detect when tabs are used and stay consistent with what's already there :)

File: apps/web/src/server/applications/case/edit/case/applications-edit-case.controller.js [hunk 2]
```diff
@@ -536,3 +548,47 @@ export async function updateApplicationsEditCaseZoomLevel(request, response) {
...
+/**
+ * View the form step for editing is material change
+ *
+ *
+ * @type {import('@pins/express').RenderHandler<ApplicationsCreateCaseIsMaterialChangeRes,
+ * {}, {}, {}, {}>}
+ */
+export async function viewApplicationsEditIsMaterialChange(request, response) {
+	if (!featureFlagClient.isFeatureActive('applics-156-material-changes')) {
+		const referrerURL = request.header('Referrer') || '/';
```
Comment (HadrianHughesKC) [inline apps/web/src/server/applications/case/edit/case/applications-edit-case.controller.js:561]: I don't think this is needed, express should handle this itself. Unless there was a reason you needed to add it.

File: apps/web/src/server/views/applications/case/overview-legacy.njk [hunk 3]
```diff
@@ -55,9 +55,9 @@
       }
```
Comment (francescobrenci-AR) [inline apps/web/src/server/views/applications/case/overview-legacy.njk:55]: we should have the `Material change` field displayed based on ht eFF status in the overview-legacy as well

File: apps/e2e/cypress/support/utils/createProjectInformation.js [hunk 4]
```diff
@@ -121,6 +122,7 @@ export const projectInformation = (options = {}) => {
 		internalDateFullFormatted,
 		internalDateMonth,
 		internalDateYear,
+		isMaterialChange,
```
Comment (francescobrenci-AR) [inline apps/e2e/cypress/support/utils/createProjectInformation.js:125]: We should add tests to validate the `Material change` field exists  and to validate the ability to change the value when when `isMaterialchange` is true -  they can run when the FF is ON

**Review by francescobrenci-AR — CHANGES_REQUESTED** [review]

File: apps/web/src/server/applications/case/edit/case/applications-edit-case.controller.js (hunk 2 above)
Comment (HadrianHughesKC) [reply apps/web/src/server/applications/case/edit/case/applications-edit-case.controller.js:561]: Ignore this, I misread the code 👍

File: apps/web/src/server/views/applications/case/overview-legacy.njk (hunk 3 above)
Comment (christopher-semmens) [reply apps/web/src/server/views/applications/case/overview-legacy.njk:55]: @francescobrenci-AR I might be mistaken as I thought when the 'applic-55-welsh-translation' flag is false but 'applics-156-material-changes' is true the 'is material change' field appears on the 'project information' page. If 'applic-55-welsh-translation' is true it then appears on the new overview page. Is it your understanding that this is not the case?

File: apps/e2e/cypress/support/utils/createProjectInformation.js (hunk 4 above)
Comment (christopher-semmens) [reply apps/e2e/cypress/support/utils/createProjectInformation.js:125]: @francescobrenci-AR I left this variable in the codebase by mistake. This e2e journey involves the creation of the case. It looks like currently there are no e2e tests for the edit fields and pages.

File: apps/web/src/server/views/applications/case/overview-legacy.njk (hunk 3 above)
Comment (francescobrenci-AR) [reply apps/web/src/server/views/applications/case/overview-legacy.njk:55]: yes - that's correct, that is the expected behaviour

**Review by francescobrenci-AR — DISMISSED** [review]

**Review by francescobrenci-AR — APPROVED** [review]

**Pull request merged**